### DISCUSSION TITLE
ValueError: Cluster table must include column named "metacluster"

### DISCUSSION THREAD

**Discussion by c0urtneyt** [angelolab/ark-analysis#902] [Q&A]
Hello,

I am running through 2_Cluster_Pixels, and have been able to get through the code previously. Today I came across this error at 3: Visualizing Results:
![clustertable_error](https://user-images.githubusercontent.com/40770583/217338175-651f936f-3ba9-4fe5-996d-93d9c1ff81ab.PNG)

I went through to locate the folder containing the error and it contains these files:
![metacluster_remap_gui](https://user-images.githubusercontent.com/40770583/217338523-b3f0cf81-b54a-4777-986b-10b4571ed533.PNG)

Do you have any advice? 
Thank you so much in advance!

**Comment by ngreenwald** [comment]
@alex-l-kong is this related to any of the recent changes?

**Reply by alex-l-kong** [reply]
@ngreenwald no, what this means is that the data @c0urtneyt is trying to input into the remapping GUI doesn't contain a `'pixel_meta_cluster'` column, which means it couldn't properly rename it to `'metacluster'`. This can only happen if the meta clustering step prior threw an error or was never otherwise completed.
